addarr.py

```diff
@@ -131,41 +139,47 @@ def main():
 def stop(update, context):
     clearUserData(context)
     context.bot.send_message(
-        chat_id=update.effective_message.chat_id, text=transcript["End"],
-        reply_markup=ReplyKeyboardRemove()
+        chat_id=update.effective_message.chat_id, text=transcript["End"]
     )
     return ConversationHandler.END
 
 
-def startSerieMovie(update, context):
-    if checkId(update):
-        if update.message.text[1:].lower() in [
-            transcript["Serie"].lower(),
-            transcript["Movie"].lower(),
-        ]:
-            logger.debug(
-                f"User issued {update.message.text} command, so setting user_data[choice] accordingly"
-            )
-            context.user_data.update(
-                {
-                    "choice": transcript["Serie"]
-                    if update.message.text[1:].lower() == transcript["Serie"].lower()
-                    else transcript["Movie"]
-                }
-            )
-        elif update.message.text.lower() == transcript["New"].lower():
-            logger.debug("User issued New command, so clearing user_data")
-            clearUserData(context)
+def startSerieMovie(update : Update, context):
+    if not checkId(update):
         context.bot.send_message(
-            chat_id=update.effective_message.chat_id, text=transcript["Title"]
+            chat_id=update.effective_message.chat_id, text=transcript["Authorize"]
         )
         return SERIE_MOVIE_AUTHENTICATED
+
+    if update.message is not None:
+        reply = update.message.text.lower()
+    elif update.callback_query is not None:
+        reply = update.callback_query.data.lower()
     else:
-        context.bot.send_message(
-            chat_id=update.effective_message.chat_id, text=transcript["Authorize"]
-        )
         return SERIE_MOVIE_AUTHENTICATED
 
+    if reply in [
```

Comment:
Can only be equal if compared from [1:] not from [0:], because `/` isn't in the transcript.